src/main/java/com/example/bookshop/securityconfig/jwt/JwtUtil.java

```diff
@@ -0,0 +1,59 @@
+package com.example.bookshop.securityconfig.jwt;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.function.Function;
+import javax.crypto.SecretKey;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtUtil {
+    private final SecretKey secret;
+    private final Long expiration;
+
+    public JwtUtil() {
+        String secretString = "serg123456789qwrtyuiopasdfghjklzxcvbnm";
+        expiration = 300000L;
+        secret = Keys.hmacShaKeyFor(secretString.getBytes(StandardCharsets.UTF_8));
+    }
```

Comment:
```suggestion
    public JwtUtil(@Value(value = "${jwt.secret}") String secretString) {
        secret = Keys.hmacShaKeyFor(secretString.getBytes(StandardCharsets.UTF_8));
    }
```

Reply:
When I initialise in the constructor @Value(value = "${jwt.secret}", I have BeanInjection Exception and I don't know why? So I used straight initialisation in the constructor.

Reply:
Correct, let's use straight initialisation in the constructor